src/styles/themes/satellite.scss

```diff
@@ -1066,6 +1066,14 @@ $break-medium: 767px;
   font-weight: normal;
 }
 
+/**
+ * PoweredBy
+ */
+
+.ais-PoweredBy-logo {
+  margin-left: 0.3rem;
```

Comment:
maybe it should be moved to the reset theme from the Algolia theme?

Reply:
So both themes include it, it makes sense